comps/comps-foreman-plugins-el8.xml

```diff
@@ -183,6 +183,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
+      <packagereq type="default">rubygem-acts_as_list-doc</packagereq>
```

Comment:
the tooling added this, i assume it was missed previously, i can remove it if desired